projects/storefrontapp/server.ts

```diff
@@ -44,7 +44,7 @@ export function app() {
   // Our Universal express-engine (found @ https://github.com/angular/universal/tree/master/modules/express-engine)
   server.engine(
     'html',
-    ngExpressEngine({
+    cxExpressEngine({
       bootstrap: AppServerModule,
     })
```

Comment:
[QUESTION]
Do you know by any chance, if our `cxExpressEngine` is capable of accepting something other than `AppServerModule`, for instance, a `bootstrap()` function returned by the new "standalone" format in the `main.server.ts` file?
Why I ask: in fresh ng17 apps what you get exported from `main.server.ts` is:
![image](https://github.com/SAP/spartacus/assets/4001059/108f66a1-ae9c-40f5-9e4b-a3439cba2fd2)
And then `bootstrap` is passed to the `CommonEngine.render()` function:
![image](https://github.com/SAP/spartacus/assets/4001059/1d559361-d873-48a1-b1f7-daa8b60b16ac)

Shall we check it additionally?

Reply:
I've tested it and it should work. If new `CommonEngine` and related interfaces support both `bootstrap()` and `NgModule`, so do we.